### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by lilyfrancis**
In this section of the App, I added styling to the Bookstore Application.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -1,41 +1,33 @@
...
+        </ul>
+      </div>
+      <div className="d-flex">
+        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="currentColor" className="bi bi-person-fill rangi" viewBox="0 0 16 16">
+          <path d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1H3zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6z" />
+        </svg>
+      </div>
+    </div>
+  </nav>
+);
```
Comment (omar-labana) [inline src/components/Navbar.js]: - [ ] The NavBar should match the rest of the layout and it should have a different background color, Kindly update your head to match the Zeplin design.
image

File: src/components/Book.js [hunk 2]
```diff
@@ -12,10 +12,34 @@ const Book = ({ book }) => {
...
+          </div>
+        </div>
+        <div className="d-flex row details">
+          <span className="col currentChapter">Current Chapter</span>
+          <span className="chapter currentLesson "> 176 </span>
+          <div className="col">
+            <button type="submit" className="bg-primary update-btn"> Update Progress</button>
+          </div>
+        </div>
+
```
Comment (omar-labana) [inline src/components/Book.js]: - [ ] Horizontal positioning of each `Book Category` and `Book Title` is a bit off. Also, the white space between the `progress` and `Chapter info` is a bit large, kindly update your `book` component to match the Zeplin design.

Zeplin design
image

- [ ] Each `book` is missing the `Author` name. Also, the vertical separators `|` between `Comment`, `Remove`, and `Edit` are missing, Kindly update your `book` component to match the Zeplin design(text colors are not matching).
Zeplin design
image

File: src/components/AddBook.js [hunk 3]
```diff
@@ -22,12 +22,20 @@ const AddBook = () => {
...
+          </div>
+          <div className="col-lg-4 p-2">
+            <input className="form-control" placeholder="Add Category" value={category} type="text" onChange={(e) => setCategory(e.target.value)} />
+          </div>
+          <div className="col-lg-4 p-2 d-grid">
+            <button type="submit" className="bg-primary text-white"> Add Book</button>
+          </div>
+        </div>
       </div>
     </form>
```
Comment (omar-labana) [inline src/components/AddBook.js]: - [ ] The form responsible for adding a new book isn't properly styled (input fields and the horizontal line before the form), Kindly update your book component to match the Zeplin design.
Zeplin design
image

- [ ] Kindly, update the text of the `title` and `category` input fields to be `Book title` and `Category` respectively.
- [ ] Kindly, update the text of the submit button to `ADD BOOK`.
- [ ] The `Category` input should be a `select` field with options of the previously given categories.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @lilyfrancis, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by er-contreras — APPROVED** [review]
Hi @lilyfrancis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**